dependencies.props

```diff
@@ -38,7 +39,7 @@
     <XunitPackageVersion>2.2.0-beta2-build3300</XunitPackageVersion>
     <XunitConsoleNetcorePackageVersion>1.0.2-prerelease-00177</XunitConsoleNetcorePackageVersion>
     <XunitPerformanceApiPackageVersion>1.0.0-beta-build0012</XunitPerformanceApiPackageVersion>
-    <MicrosoftDiagnosticsTracingTraceEventPackageVersion>1.0.3-alpha-experimental</MicrosoftDiagnosticsTracingTraceEventPackageVersion>
+    <MicrosoftDiagnosticsTracingTraceEventPackageVersion>2.0.2-dev</MicrosoftDiagnosticsTracingTraceEventPackageVersion>
```

Comment:
Does this need to be pushed to a feed before this PR can be merged?  I don't see it on MyGet.

Reply:
Yes, the new TraceEvent package will need to be added to MyGet first

Reply:
Ok, do we have a package ready for this?  If so, let's confirm with @vancem, and then get it pushed (I can help do this).